Misc/reputations.json

```diff
@@ -124,18 +124,6 @@
         "CBEvents"
       ]
     },
-    {
-      "id": "cveScore",
-      "version": -1,
-      "regex": "CVE-\\d{4}-\\d{4,7}",
-      "reputationScriptName": "cveReputation",
-      "details": "CVE CVSS Score",
-      "contextValue": "ID",
-      "contextPath": "CVE(val.ID && val.ID === obj.ID)",
-      "enhancementScriptNames": [
-        "cveSearch"
-      ]
-    },
```

Comment:
the `Misc/reputations.json` is for prior to 5.0 instances. why did you delete it?

Reply:
Because it was decided that indicator type `CVE CVSS Score` can be removed completely. The question is if we remove this old one completely then do we need to update the `fromVersion` field of the new indicator type `CVE` to reach to older server versions as well. Ani mivarer achshav.

Reply:
Spoke to @JonathanMeler. We are going to add `"toVersion": 4.9.9` to the old indicator type instead of deleting or disabling it.